doc/akri_connector/connector-metadata-schema.json

```diff
@@ -0,0 +1,343 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "aioConnectorMetadataSchemaVersion": "1.0",
+  "definitions": {
+    "schema": {
+      "type": "object",
+      "description": "The schema for a stringified JSON schema fields such as 'additionalConfigurationSchema'. Must provide inline schema xor the file that contains the schema. These schemas may be used to define the default values of unspecified fields.",
```

Comment:
For the `schemaFile` option, is this a reference to a schema that is already in schema registry, or a file from a filesystem, etc.?